app/src/main/java/com/android/unio/ui/event/overlay/PictureOverlay.kt

```diff
@@ -43,18 +67,68 @@ import kotlinx.coroutines.launch
 fun PictureOverlay(
     onDismiss: () -> Unit,
     pagerState: PagerState,
-    eventPictures: List<EventUserPicture>
+    eventPictures: List<EventUserPicture>,
+    eventViewModel: EventViewModel,
+    user: User
 ) {
+
   val scope = rememberCoroutineScope()
   val context = LocalContext.current
-  val iconSize = 40.dp
+  val arrowSize = 40.dp
+  var enableButton by remember { mutableStateOf(true) }
+  val event by eventViewModel.selectedEvent.collectAsState()
+
+  if (event == null) {
+    Log.e("PictureOverlay", "Event is null")
+    Toast.makeText(LocalContext.current, "An error occurred.", Toast.LENGTH_SHORT).show()
```

Comment:
Consider putting the good error in the Toast if you have it, here : "Event is null"